### ISSUE TITLE
Support for Centos and RHEL

### ISSUE THREAD

**Issue by ShylajaDevadiga** [rancher/rke2#85]
Node OS: Centos, RHEL

Issue:
rpms are not available. as mentioned in the issue https://github.com/rancher/rke2/issues/49

Additional info:
```
rpm -i https://rpm.rancher.io/rke2-selinux-0.1.1-rc1.el7.noarch.rpm
curl: (22) The requested URL returned error: 404 Not Found
error: skipping https://rpm.rancher.io/rke2-selinux-0.1.1-rc1.el7.noarch.rpm - transfer failed
```

**Comment by davidnuzik** [comment]
@ShylajaDevadiga we are waiting on https://github.com/rancherlabs/eio/issues/145
This is expected to not work currently.

**Comment by Oats87** [comment]
This issue should be merged into: https://github.com/rancher/rke2/issues/86

**Comment by davidnuzik** [comment]
Closing. Track in https://github.com/rancher/rke2/issues/86
